cmd/user-agent/src/pages/Interop.vue

```diff
@@ -130,7 +147,26 @@ SPDX-License-Identifier: Apache-2.0
             },
             getVP: async function () {
                 this.clearResults()
-                const credentialQuery = JSON.parse('{"web": {"VerifiablePresentation": {}}}');
+                const credentialQuery = {
+                    web: {
+                        VerifiablePresentation: {
+                            query: [
+                                {
+                                    type: "QueryByExample",
+                                    credentialQuery: {
+                                        reason: "Please present a credential for JaneDoe.",
```

Comment:
JaneDoe?

Reply:
this is test page, not exposed in dashboard